### DISCUSSION TITLE
Is There Any Way I Could Possible Have Both a Sidebar and a Bookmark Bar At The Same Time?

### DISCUSSION THREAD

**Discussion by Dorko316** [Godiesc/firefox-gx#213] [Q&A]
There are some bookmarks and extensions I'd like to keep readily available, but I don't want to clutter up the sidebar. Is there any way I can have both?

**Comment by Godiesc** [comment]
hi,
you can use the Firefox sidebar "Ctrl+B" to see your bookmarks or use the next icon to access them:

![imagen](https://github.com/user-attachments/assets/58370e95-c2cd-44e3-91a5-c3b8e53ff66b)

Right now Firefox is working on a sidebar (set `sidebar.revamp` to `true`) but I think is so baby compared to others sidebars.

**Reply by Dorko316** [reply]
Oh! I didn't know it had that! I'll have to play around with those, see what works. 

Thanks!

**Reply by Dorko316** [reply]
Yeah, you're right, it is pretty bad. I'll have to see what I can do.
